engines/dragons/cutscene.cpp

```diff
@@ -0,0 +1,888 @@
+	_vm->_actorManager->clearActorFlags(2);
+	//DisableVSyncEvent();
+	DAT_80072de8 = _vm->_actorManager->loadActor(resourceId, sequenceId, x, 199, 3);
+	//EnableVSyncEvent();
+	changeBackgroundPosition(3, param_4);
+
+}
+
+void CutScene::wideAngleEveryoneAtTable() {
+	FUN_8003d388();
+	// call_fade_related_1f();
+
+}
+
+void CutScene::FUN_8003d388() {
+	uint sequenceId;
+
+	// fade_related_calls_with_1f();
+	_vm->_actorManager->clearActorFlags(2);
+	if ((DAT_80063514 & 0x80) == 0) {
+		DAT_80072de8 = _vm->_actorManager->loadActor(0x7e,0x16,0x40,0xa0,1);
+	}
+	DAT_80072dec = _vm->_actorManager->loadActor(0x7e,0,0xbf,0xba,1);
+	if ((DAT_80063514 & 8) == 0) {
+		DAT_80072df0 = _vm->_actorManager->loadActor(0x7e,5,0x94,0x82,1);
+	}
+	if ((DAT_80063514 & 0x10) == 0) {
+		DAT_80072df4 = _vm->_actorManager->loadActor(0x7e,10,0x6f,0x95,1);
+	}
+	if ((DAT_80063514 & 4) == 0) {
+		DAT_80072df8 = _vm->_actorManager->loadActor(0x7e,0xe,0xa9,0x87,1);
+	}
+	if ((DAT_80063514 & 0x20) == 0) {
+		DAT_80072dfc = _vm->_actorManager->loadActor(0x7e,0x12,0xcd,0x8e,1);
+	}
+	if ((DAT_80063514 & 1) == 0) {
+		DAT_80072e04 = _vm->_actorManager->loadActor(0x7e,0x19,0x10e,0x89,1);
+	}
+	if ((DAT_80063514 & 2) == 0) {
+		DAT_80072e08 = _vm->_actorManager->loadActor(0x8f,2,100,0xbc,1);
+	}
+	if ((DAT_80063514 & 0x40) != 0) {
+		DAT_80072e0c = _vm->_actorManager->loadActor(0x8f,0,0xd2,100,1);
+		DAT_800830a0 = _vm->_actorManager->loadActor(0x8f,1,0xe6,0x6e,1);
+	}
+	DAT_800830b8 = _vm->_actorManager->loadActor(0xaa,0,0x2e,0x2d,1);
+	DAT_800830b8->setFlag(ACTOR_FLAG_8000);
+	DAT_800830bc = _vm->_actorManager->loadActor(0xaa,1,0x115,0x22,1);
+	DAT_800830bc->setFlag(ACTOR_FLAG_100);
+	DAT_800830bc->setFlag(ACTOR_FLAG_8000);
+	DAT_800830bc->priorityLayer = 4;
+	if ((DAT_80063514 & 0x100) != 0) {
+		DAT_800830c0 = _vm->_actorManager->loadActor(0x7e,0x1c,0x21,0x87,1);
+	}
+	if ((DAT_80063514 & 0x200) != 0) {
+		if ((DAT_80063514 & 0x800) == 0) {
+			sequenceId = 2;
+		}
+		else {
+			sequenceId = 4;
+		}
+		DAT_800830d4 = _vm->_actorManager->loadActor(0xaa,sequenceId,0xf4,199,1);
+		DAT_800830d4->setFlag(ACTOR_FLAG_8000);
+	}
+	if ((DAT_80063514 & 0x400) != 0) {
+		DAT_800830dc = _vm->_actorManager->loadActor(0xaa,3,0xf4,199,1);
+		DAT_800830dc->setFlag(ACTOR_FLAG_8000);
+	}
+	changeBackgroundPosition(0, 0);
+	_vm->waitForFramesAllowSkip(0xe);
+}
+
+
+void CutScene::closeUpKnightsAtTable() {
+	//fade_related_calls_with_1f();
+	_vm->_actorManager->clearActorFlags(2);
+	DAT_80072de8 = _vm->_actorManager->loadActor(0x7d,0,2,199,1);
+	DAT_80072dec = _vm->_actorManager->loadActor(0x81,4,2,199,1);
+	DAT_80072df0 = _vm->_actorManager->loadActor(0x81,6,2,199,1);
+	DAT_80072df4 = _vm->_actorManager->loadActor(0x81,0,2,199,1);
+	DAT_80072df8 = _vm->_actorManager->loadActor(0x81,2,2,199,1);
+	changeBackgroundPosition(1, 0);
+	_vm->waitForFrames(0xf);
+	//call_fade_related_1f();
+
+}
+
+uint16 CutScene::FUN_8003dab8(uint32 textId,uint16 x,uint16 y,uint16 param_4,int16 param_5)
+{
```

Comment:
Wrong code formatting. An opening curly brace should be at the previous line. And spaces after commas are missing